src/redisearch_rs/inverted_index/src/lib.rs

```diff
@@ -367,3 +511,6 @@ pub trait Decoder {
         }
     }
 }
+
+#[cfg(test)]
+mod tests;
```

Comment:
Any reason to not have those in the existing `/tests/` dir instead?

Reply:
Done in https://github.com/RediSearch/RediSearch/pull/6490/commits/e9640e7f0493fe063f61d386859f249d7d177902